### DISCUSSION TITLE
First Community Call

### DISCUSSION THREAD

**Discussion by rawkode** [comtrya/comtrya#210] [Announcements]
Hello!

We'll be hosting our first Comtrya community call next week! Vote now to let us know what slot works best for you and we'll pick the slot that has the most people.

https://savvycal.com/p/rawkode/2cca35e9

On this call we'll discuss:

- How YOU use Comtrya
- What we need for 1.0
- Roadmap ideas for beyond

**Comment by rawkode** [comment]
Also for discussion:

- Inputs / Outputs
- Async Runner
- Side Effects

**Reply by icepuma** [reply]
A thing I want to discuss is `Variable Inputs` like `doppler://`

**Reply by rawkode** [reply]
`comtrya watch`

**Reply by rawkode** [reply]
CLI output and TUIs
